src/editor/Toolbox.tsx

```diff
@@ -12,7 +12,7 @@ export interface PreviewNodeProps<S extends Schema> {
 
 export function PreviewNode<S extends Schema>(props: PreviewNodeProps<S>) {
 	const context = useContext(CanvasContext)
-	const { borderColor, toolBoxImgWidth } = context.options
+	const { borderColor, textColor, toolBoxImgWidth } = context.options
```

Comment:
Nit: In most cases we pull colours from the MUI theme. 
```
const theme = useTheme();
const textColor = theme.palette.common.black;
```

Reply:
Agree that makes sense, think it could be a separate PR to pull colors from MUI theme and use them everywhere. Let me know what you think